_appmap/web_framework.py

```diff
@@ -145,7 +147,7 @@ def after_request_main(self, rec, status, headers, start, call_event_id) -> None
             parent_id=call_event_id,
             elapsed=duration,
             status_code=status,
-            headers=dict(headers.items()),
```

Comment:
The keys in a headers collection need to be matched in a case-insensitive manner, so we can't just create a `dict` from them. This also isn't necessary, because the various web frameworks represent them with with a type that quacks enough like a `dict` for our purposes.